src/libraries/Common/src/System/Security/Cryptography/SLHDsa.cs

```diff
@@ -0,0 +1,1620 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Formats.Asn1;
+using System.Security.Cryptography.Asn1;
+
+// The type being internal is making unused parameter warnings fire for
+// not-implemented methods. Suppress those warnings.
+#pragma warning disable IDE0060
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    ///   Represents an SLH-DSA key.
+    /// </summary>
+    /// <remarks>
+    ///   Developers are encouraged to program against the <c>SLHDsa</c> base class,
+    ///   rather than any specific derived class.
+    ///   The derived classes are intended for interop with the underlying system
+    ///   cryptographic libraries.
+    /// </remarks>
+    [Experimental(Experimentals.PostQuantumCryptographyDiagId)]
+    internal abstract partial class SLHDsa : IDisposable
```

Comment:
The capitalization rules on things are hard.  I expect what we want is `SlhDsa`, since it is two three letter abbreviations concatenated.